font/src/darwin/mod.rs

```diff
@@ -278,6 +281,31 @@ pub struct Font {
 
 unsafe impl Send for Font {}
 
+
+/// Set subpixel anti-aliasing on macOS.
+///
+/// Sub-pixel anti-aliasing has been disabled since macOS Mojave by default. This function allows
+/// overriding the global `CGFontRenderingFontSmoothingDisabled` setting on a per-application basis
+/// to re-enable it.
+///
+/// This is a no-op on systems running High Sierra or earlier (< 10.14.0).
+pub fn set_font_smoothing(enable: bool) {
+    let min_macos_version = NSOperatingSystemVersion::new(10, 14, 0);
+    unsafe {
+        // Check that we're running at least Mojave (10.14.0+).
+        if !NSProcessInfo::processInfo(nil).isOperatingSystemAtLeastVersion(min_macos_version) {
+            return
+        }
```

Comment:
@chrisduerr whats your/our preferred way here? `!` or ` == false`?

Reply:
No point to ever do `== false` really.